packages/angular/projects/clr-angular/src/forms/combobox/combobox.html

```diff
@@ -82,6 +82,7 @@
     type="button"
     class="clr-combobox-trigger"
     tabindex="-1"
+    [disabled]="control?.disabled || null"
```

Comment:
A test won't be bad for that :)

Reply:
This seems impossible to test... 

Tried so many ways to check if the button is `disabled`, has an attribute `disabled` with `true`, only has an attribute `disabled`... nothing. Visually works as expected. I am out of ideas...

Reply:
`fakeAsync` for the win. Thank you!

Reply:
🍺